src/cell.cpp

```diff
@@ -779,7 +779,7 @@ CSGCell::contains_complex(Position r, Direction u, int32_t on_surface) const
 // DAGMC Cell implementation
 //==============================================================================
 #ifdef DAGMC
-DAGCell::DAGCell() : Cell{} {};
+DAGCell::DAGCell() : Cell{} { simple_ = true; };
```

Comment:
Is this always going to be true for DAGMC cells? Would this be better in the header? philisophically speaking that is

Reply:
I thought about that too, but because it's an attribute of the abstract `Cell` class that would make the default true for `CSGCells` as well. This value is initialized in `CSGCell`'s constructor as `true` though, so perhaps it would be alright. 

https://github.com/openmc-dev/openmc/blob/327ace77a59069e3b9972cf31e0b706e59d30cfd/src/cell.cpp#L404-L412


Thoughts @paulromano?

Reply:
If we were to assign a value in the header, it should be a default that is safe (false). If we did that, we'd still have to assign a value for `DAGCell` in the .cpp file since it would be different.